pom.xml

```diff
@@ -129,7 +129,7 @@
                             <alias>elasticsearch</alias>
                             <name>${dockerCafDataProcessingOrg}policy-elasticsearch-container${dockerProjectVersion}</name>
                             <build>
-                                <from>cafinternal/prereleases:opensuse-elasticsearch2-1.0.0-SNAPSHOT</from>
+                                <from>cafapi/opensuse-elasticsearch2:1.0.0</from>
```

Comment:
We should be using the 2 part version number for base images so that any patches get automatically picked up. So this should be cafapi/opensuse-elasticsearch2:1.0

Reply:
Ok, Updated the pom file to use 1.0